charts/kube-prometheus-stack/values.yaml

```diff
@@ -1227,6 +1227,23 @@ kubeApiServer:
     # - targetLabel: __address__
     #   replacement: kubernetes.default.svc:443
 
+    ## Enable scraping Service Level Indicator (SLI) metrics (/metrics/slis)
+    slis: false
+
+    ## SLIs MetricRelabelConfigs to apply to samples after scraping, but before ingestion.
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig
+    ##
+    slisMetricRelabelings: []
+
+    ## SLIs RelabelConfigs to apply to samples before scraping
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig
+    ##
+    slisRelabelings:
+      - sourceLabels: [__metrics_path__]
+        targetLabel: metrics_path
+    # - sourceLabels: [__meta_kubernetes_pod_node_name]
+    #   targetLabel: nodename
+
```

Comment:
I agree, but since kubelet's `ServiceMonitor` was structured this way, I had no choice but to adapt it.
I will change the structure as you suggested except for kubelet, but should I also change kubelet to this different structure only for `/metrics/slis`?

Reply:
> I will change the structure as you suggested except for kubelet

Pushed.  (No modifications have been made for kubelet.)